src/services/lansweeper/baseRequest.ts

```diff
@@ -1,30 +1,31 @@
 import { isEmpty } from "lodash";
 import { proxyFetch, adminGenericProxyFetch } from "@deskpro/app-sdk";
-import { BASE_URL, placeholders } from "../../constants";
-import { getQueryParams, getRequestBody } from "../../utils";
+import { GRAPHQL_URL, placeholders } from "../../constants";
+import { getQueryParams, getRequestBody, parseGlobalAccessToken } from "../../utils";
 import { LansweeperError } from "./LansweeperError";
 import type { Request, FetchOptions } from "../../types";
 
 const baseRequest: Request = async (client, {
   url,
   rawUrl,
   data,
-  method = "GET",
+  method = "POST",
   queryParams = {},
   headers: customHeaders,
   settings,
 }) => {
   const dpFetch = await (!settings ? proxyFetch : adminGenericProxyFetch)(client);
-  const baseUrl = rawUrl ? rawUrl : `${BASE_URL}${url || ""}`;
+  const baseUrl = rawUrl ? rawUrl : `${GRAPHQL_URL}${url || ""}`;
   const params = getQueryParams(queryParams);
   const body = getRequestBody(data);
+  const tokens = parseGlobalAccessToken(settings?.global_access_token);
 
   const requestUrl = `${baseUrl}${isEmpty(params) ? "": `?${params}`}`;
   const options: FetchOptions = {
     method,
     body,
     headers: {
-      "Authorization": `Bearer ${placeholders.ACCESS_TOKEN}`,
+      "Authorization": `Bearer ${tokens.access_token || placeholders.ACCESS_TOKEN}`,
```

Comment:
To reduce complexity, it'd be nice to not have this fallbacks. Either the calling code passes things in correctly, or it should fail. If fallbacks want to be used, then it's up to the calling code to define that.

Reply:
This is not a fallback. It's an opportunity to access to settings in the admin side, or use placeholders in the app on the agent side

Reply:
Maybe I'm reading it wrong but it looks like it uses `tokens.access_token` and if that's not there, it falls back to use the `placeholders.ACCESS_TOKEN`. My suggestion was to lift this logic out of here and put it in the calling code instead?

Reply:
I don't want to complicate the interface of function. Changed to this
```
const isAdmin = !settings;
...
"Authorization": `Bearer ${isAdmin ? tokens.access_token : placeholders.ACCESS_TOKEN}`,
```

So it's clearer what's going on?